roles/foreman_installer_devel_scenario/templates/katello-devel-answers.yaml

```diff
@@ -35,5 +35,5 @@ foreman_proxy::plugin::dynflow: true
 foreman_proxy::plugin::monitoring: false
 foreman_proxy::plugin::omaha: false
 foreman_proxy::plugin::openscap: false
-foreman_proxy::plugin::remote_execution::ssh: true
+foreman_proxy::plugin::remote_execution::script: true
```

Comment:
this will break older versions so I think I need to use the templating to make change only when `if foreman_provisioning_foreman_version == 'nightly'`

Reply:
*devel* ;)